sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// an Extract Summary action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an Extract Summary action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
```

Comment:
I noticed we have another constructor that takes an `Options` type in other Actions, but I wasn't sure if we should add it here since we don't have a separate API for extractive summarization (we would have an options type only for this constructor). I think it depends on why we decided to add those helper constructors in the first place.

Reply:
Great question.
so we only did that for the actions that had an equivalent non LRO method and an existing `xxOptions` type. It is a convenience in case a developer started in the non LRO world and then wants to come here. They don't need to manually pass all properties, they can just pass the object they already created/configured.
You are right that in this case, we don't need it